util/fipstools/delocate/delocate.go

```diff
@@ -488,6 +490,19 @@ func (d *delocation) processAarch64Instruction(statement, instruction *node32) (
 		}
 
 		return d.loadAarch64Address(statement, targetReg, symbol, offset)
+	case "bl":
```

Comment:
If I understand the solution here:
* we found some `bl` instructions that jumped to accessor functions that have the suffix `_bss_get` as a suffix.
* but these functions were not found because they would have been formed by delocate.go at https://github.com/awslabs/aws-lc/blob/d3cbac9bd889592e83d94c3c16282c320b4d8df4/util/fipstools/delocate/delocate.go#L205 when a `.comm` directive was found and they were added to `bssAccessorsNeeded`.
* so we now find them where they're called in `bl` instructions and add them to that same array.

If this understanding is correct, is `bl` the only instruction they're found with?